app/views/events/forms/_League.html.erb

```diff
@@ -1,9 +1,19 @@
+<%= render "events/forms/Dates", f: form%>
```

Comment:
I don't think you have to put an absolute path here, just Dates should be enough.